package-lock.json

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
What it this file for?

Reply:
@dogi I am not sure where I got it. But I have deleted it.